meshtastic/tests/test_slog_power_logger.py

```
@@ -139,6 +160,35 @@ def test_root_dir_legacy_alias_warns_once(
+
+    try:
+        monkeypatch.setattr(typing, "TYPE_CHECKING", True)
+        monkeypatch.setattr(pa, "DataType", _SubscriptableDataType)
+        reloaded = importlib.reload(slog_module)
+        assert reloaded.ArrowDataType is _SubscriptableDataType
+    finally:
+        monkeypatch.setattr(typing, "TYPE_CHECKING", original_type_checking)
+        monkeypatch.setattr(pa, "DataType", original_data_type)
+        try:
+            importlib.reload(slog_module)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)